mcs/build/tests.make

```diff
@@ -157,6 +157,9 @@ $(test_makefrag): $(test_response)
 
 -include $(test_makefrag)
 
+build-test-lib: $(test_lib)
```

Comment:
Would be better to teach tests about PROFILE so standard `PROFILE=xx make check` works